commit 2dfae0b2a49d311c75470a981ca7e7f5c0f22d3e
Author: bertin <macportsuser@macports.org>
Date:   Fri May 23 21:28:47 2025 +0200

    /opt/local/site-ports/sysutils/MacPorts-pextlib/files/lamarck/patch-aclocalm4-libcurl.diff

diff --git a/aclocal.m4 b/aclocal.m4
index cc92d20c..c27f1681 100644
--- a/aclocal.m4
+++ b/aclocal.m4
@@ -887,15 +887,7 @@ AC_DEFUN([MP_LIBCURL_FLAGS],[
 	fi
 
 	CFLAGS_LIBCURL=$($CURL_CONFIG --cflags)
-	if test "x$curlprefix" = "x"; then
-		# System curl-config emits absurd output for --libs
-		# See rdar://7244457
-		LDFLAGS_LIBCURL="-lcurl"
-	else
-		# Due to a bug in dist, --arch flags are improperly supplied by curl-config.
-		# Get rid of them.
-		LDFLAGS_LIBCURL=$($CURL_CONFIG --libs | [sed 's/-arch [A-Za-z0-9_]* //g'])
-	fi
+	LDFLAGS_LIBCURL=$($CURL_CONFIG --libs)
 
 	AC_SUBST(CFLAGS_LIBCURL)
 	AC_SUBST(LDFLAGS_LIBCURL)
